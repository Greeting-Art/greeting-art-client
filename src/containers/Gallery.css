.galleryPage {
  display: grid;
  grid-template-rows: 15vh 85vh;
}

.galleryHeader {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  font-size: 1.8em;
}

.galleryBody {
  display: flex;
  padding-left: 125px;
}

.controlBar {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
}

.galleryBar {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
}

.centerColumn {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  background-color: rgb(230, 227, 227);
}

.canvasWrapper {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}

.lowerBar {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
}

.galleryLI {
  list-style-type: none;
}

.loadingSpinner {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  background-image: url('../assets/appIcons/splatter.png');
  background-repeat: no-repeat;
  background-position: center;
  background-size: contain;
  height: 50vh;
}
