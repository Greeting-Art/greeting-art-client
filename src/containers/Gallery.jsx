/* eslint-disable indent */
import React, { useState } from 'react';

export default function Gallery() {
  const [loading, setLoading] = useState(true);

  if (loading) return <h2>loading...</h2>;

  return (
    <div>
      <h1>Gallery</h1>
    </div>
  );
}
