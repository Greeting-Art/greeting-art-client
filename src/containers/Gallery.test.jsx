/**
 * @jest-environment jsdom
 */
import React from 'react';
import { render, screen } from '@testing-library/react';
import Gallery from './Gallery';

describe('art gallery container', () => {
  it('displays a list of pieces of art', async () => {
    render(<Gallery />);

    // const screenLoading = await screen.getByText('loading...');
    // expect(screenLoading).toMatchSnapshot();

    const toECardButton = await screen.getByText('Send Art as eCard');
    expect(toECardButton).toMatchSnapshot();
  });
});
