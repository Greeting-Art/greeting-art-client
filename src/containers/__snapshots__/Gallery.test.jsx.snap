// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`art gallery container displays a list of pieces of art 1`] = `
<h2>
  loading...
</h2>
`;
